src/tui.rs

```diff
@@ -2289,7 +2289,8 @@ impl<'a> Context<'a, '_> {
                             let trackable = track_rect.height() - tc.thumb_height;
                             let delta_y = mouse.y - self.tui.mouse_down_position.y;
                             tc.scroll_offset.y = tc.scroll_offset_y_drag_start
-                                + (delta_y as f64 / trackable as f64 * scrollable_height as f64) as CoordType;
+                                + (delta_y as i64 * scrollable_height as i64 / trackable as i64)
+                                    as CoordType;
```

Comment:
As mentioned in #315, I'd like to use floats here once everything uses floats. I think that makes sense long-term. For now, I'd like to stick with `i64` for intermediate results for consistency.